.ci/unit-tests/gcb-membership-checker-test.yml

```diff
@@ -0,0 +1,10 @@
+---
+steps:
```

Comment:
Not a blocker / could be done in a separate PR or as part of a later refactor.

Reply:
I added it as a cloud build trigger since we need file filter and security guard (/gcbrun needed) for the tests to be triggered. I'll have to check if github action have the similar features and we can move it to github action in a separate PR later.